ingestor-default/src/main/scala/za/co/absa/hyperdrive/ingestor/implementation/writer/mongodb/MongoDbStreamWriterAttributes.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hyperdrive.ingestor.implementation.writer.mongodb
+
+import za.co.absa.hyperdrive.ingestor.api.writer.StreamWriterCommonAttributes
+import za.co.absa.hyperdrive.ingestor.api.{HasComponentAttributes, PropertyMetadata}
+
+trait MongoDbStreamWriterAttributes extends HasComponentAttributes {
+  private val rootFactoryConfKey = "writer.mongodb"
+  val KEY_URI = s"$rootFactoryConfKey.uri"
+  val KEY_DATABASE = s"$rootFactoryConfKey.database"
+  val KEY_COLLECTION = s"$rootFactoryConfKey.collection"
+  val KEY_EXTRA_CONFS_ROOT = s"$rootFactoryConfKey.options"
+
+  override def getName: String = "Parquet Stream Writer"
```

Comment:
Good spot, thanks!

Reply:
Why you don't like the idea? This looks like it can help with UI integration

Reply:
First, it's very hard to predict whether this feature will ever be used. Second, it was a nice idea to enforce documentation, but in practice, there is no way to enforce good and meaningful documentation, and so user-facing documentation should probably sit in the DB or somewhere, where it can be modified very quickly, without rebuilding or redeploying the app.